### PR TITLE
FPO-282

### PR DESCRIPTION
**Pull request by Lee365**
### Jira link
https://transformuk.atlassian.net/browse/FPO-282

### What?

I have added/removed/altered:

- [x] Added
- [x] Removed
- [x] Altered

### Why?

I am doing this because:

-

### AC

- Tested
- Shared with stakeholders
- Pair reviewed

### CODE REVIEW

File: src/controllers/verificationController.ts [hunk 1]
```diff
@@ -17,5 +17,20 @@ export const checkVerificationDetails = async (req: Request, res: Response): Pro
 }
 
 export const applicationComplete = async (req: Request, res: Response): Promise<void> => {
-  res.render('completion')
+  const body = req.body
+
+  try {
+    const termsList = req.body.terms as string[]
+    const _termsList = (typeof termsList === 'string') ? [termsList] : termsList
+    console.debug('Terms count:', _termsList?.length)
```
Comment (mkayad) [inline src/controllers/verificationController.ts:25]: I think we can remove the log here,

File: src/controllers/verificationController.ts [hunk 2]
```diff
@@ -17,5 +17,20 @@ export const checkVerificationDetails = async (req: Request, res: Response): Pro
...
+
+    if (_termsList?.length > 3) {
```
Comment (mkayad) [inline src/controllers/verificationController.ts:27]: for the validation, did you look into using this module express-validator?

Comment (Lee365) [reply src/controllers/verificationController.ts:27]: Yes - good idea.

File: src/controllers/verificationController.ts (hunk 1 above)
Comment (Lee365) [reply src/controllers/verificationController.ts:25]: Sure

**Review by mkayad — APPROVED** [review]
LGTM

**Pull request merged**